src/components/landing.js

```diff
@@ -38,27 +38,37 @@ class Landing extends Component {
       <div className="wrap">
          <section>
             {/* <Logo /> */}
-            <div style={{display: 'flex', flexDirection: 'row', alignItems: 'center'}}>
+            <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
                <h1>HackUNT 2021</h1>
                {/* <p><a href='/2020_Sponsorship_Doc.pdf'>interested in sponsoring?</a></p> */}
             </div>
             <div style={{ width: 100, height: 20 }} className="bg-green"></div>
 
             {/* {panels.filter((v, i) => parseInt(i) <= this.state.numberToShow)} */}
-            <div style={{display: 'flex', flexDirection:'row'}}>
-               <div style={{width: '50%'}}>
-                  <h3 style={{fontSize: '2rem'}}>March 13 - 14, 2021</h3>
-                  <h3 style={{fontSize: '1.5rem'}}><span role="img" aria-label="construction">🚧</span> Under Construction <span role="img" aria-label="construction">🚧</span></h3>
-                  <p style={{marginTop: '5rem'}}>
-                     HackUNT is the University of North Texas' annual hackathon. 
-                     A hackathon is a 24-hour development competition for coders, designers, and anyone with a good idea! 
-                     Don't worry if you aren't experienced either, HackUNT is a great place to learn! 
-                     We will have mentors and professionals from the industry to help along the way. 
+            <div style={{ display: 'flex', flexDirection: 'row' }}>
+               <div style={{ width: '50%' }}>
+                  <h3 style={{ fontSize: '2rem' }}>March 13 - 14, 2021</h3>
+                  <h3 style={{ fontSize: '1.5rem' }}>
+                     <span role="img" aria-label="construction">🚧</span>
+                      Under Construction
+                      <span role="img" aria-label="construction">🚧</span>
+                  </h3>
+                  <p style={{ marginTop: '1rem' }}>
+                     HackUNT is the University of North Texas' annual hackathon.
+                     A hackathon is a 24-hour development competition for coders, designers, and anyone with a good idea!
+                     Don't worry if you aren't experienced either, HackUNT is a great place to learn!
+                     We will have mentors and professionals from the industry to help along the way.
                      <br />
                   </p>
-                  <MailingListSignup />
+                  {/*Removed as we are no longer allowing mailing list signups in favor of having a register <MailingListSignup /> */}
+                  <p>
+                     <a href="https://hackunt.typeform.com/to/DlhVc6de" target="_blank" rel="nooperer noreferrer">
+                        <button style={{ width: "45%", height: "5em", marginRight: "10%" }}>Register</button>
```

Comment:
Can we make the font bolder? Also, let's take the sponsor button off as it doesn't do anything

Reply:
yeah